docs/stransmit.md

```diff
@@ -164,15 +164,21 @@ example, **-t:60**. Alternatively you can also separate them by a space,
+- **-bandwidth, -bitrate, -b** [**To be added**] - slow down sending to not send more data than given bandwidth (in bits per second). Note that it has nothing to do with SRT-controlled bandwidth; rather a possibility to have a "live-alike" reading speed from an infinite-speed input source (such as file). Better to think of it as for "testing purposes" because the real live transmission requires its speed synchronized with TS timestamps and *srt-live-transmit* has no ability to interpret them.
 - **-verbose, -v** - display additional information on the standard output. Note that it's not allowed to be combined with output specified as **file://con**
-- **-bandwidth-report, -bitrate-report, -report, -r** - Throw periodic bandwidth reports on the standard output. The value is the number of sending events between reports.
-- **-crash, -k** - Force the program to "crash" (terminate abnormally with core dump) when a problem occurred (even such as connection got broken). This is useful only for developers.
+- **-bandwidth-report, -bitrate-report, -report, -bwreport, -r** - Show periodic bandwidth reports on the standard output. The value is the number of sending events between reports.
+- **-stats", -stats-report-frequency, -s** - Output periodic SRT statistics reports to the standard output or file (see **-statsout**).
+- **-statsout"** - SRT statistics output: standard output (**stdout**) or filename.
+- **-pf, -statspf** - SRT statistics print format. Values: **json**, **csv**, **default**.
+- **-crash, -k** [**To be added**] - Force the program to "crash" (terminate abnormally with core dump) when a problem occurred (even such as connection got broken). This is useful only for developers.
 - **-loglevel** - lowest logging level for SRT, one of: *fatal, error, warning, note, debug* (default: *error*)
 - **-logfa** - selected FAs in SRT to be logged (default: all is enabled, that is, you can filter out log messages comong from only wanted FAs using this option)
-- **-ttl, -max-loss-delay** - use `SRTO_LOSSMAXTTL` option for all SRT medium (applies delayed sending of loss report, if UDP packet reordering is detected, in the number of packets to be received after the lost one before reporting a loss; the real "reorder tolerance" value is controlled internally, starts from 0, and grows only upon detected packet reordering; this option only defines its maximum value; the sensible value is 10)
+- **-ttl, -max-loss-delay** [**To be added**] - use `SRTO_LOSSMAXTTL` option for all SRT medium (applies delayed sending of loss report, if UDP packet reordering is detected, in the number of packets to be received after the lost one before reporting a loss; the real "reorder tolerance" value is controlled internally, starts from 0, and grows only upon detected packet reordering; this option only defines its maximum value; the sensible value is 10)
```

Comment:
Don't add this, rather remove this description. It was probably short living and then removed; this can be well enough used with `lossmaxttl` option.

Reply:
Deleted.